### ISSUE TITLE
Multiple events per day?

### ISSUE THREAD

**Issue by xbroak** [SundeepK/CompactCalendarView#10]
Say my database has 4 items for a particular date,

I'll addEvent() for the date, then what happens when the item is clicked? Nothing? I presume it will just add the indicator and displaying the items is up to me? If so, how do you recommend doing this? Storing and searching by date and displaying them once tapped?

**Comment by SundeepK** [comment]
I didn't want to give CompactCalendarView the responsibility of managing events. Only displaying them to the UI. CompactCalendarView exposes a listener for when a day is clicked with the date. That should give you enough info. I would query the database, store the data you get back in some local cache, maybe Map<Long, SomeEvent>. Then you can get events based on the start of each day and display. 

I don't know if I would add this functionality, but I'm uncertain yet. Maybe a callback which gives back a date and a list of events. It shouldn't to be to hard just some people might want to manage the data their own way using some cache which expires with different configurations. Adding something like this would make things much more complex.

TL;DR
You must manage your own data and CompactCalendarView will display it for you.

**Comment by SundeepK** [comment]
I know this issue is quite old, but I've implemented this functionality in #52. Have a look, you can add a list of events for a particular day and also add some custom data along with each event. I will be releasing a beta for this very soon.

**Comment by SundeepK** [comment]
I'll close this issue since I am tracking changes in #52 instead.
